recipes/Dessert/frozen_chocolate_cake.md

```diff
@@ -0,0 +1,38 @@
+# Frozen Chocolate Cake
+(not frikin pudding)
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-10%25-brightgreen.svg)
+![Speed Badge](https://img.shields.io/badge/Speed-10min-brightgreen.svg)
```

Comment:
It is edible after the 10 min and would taste great but it's summer and it's better frozen :wink: